solutions/standard/variables.tf

```
@@ -16,19 +16,19 @@ variable "use_existing_resource_group" {
 
 variable "resource_group_name" {
   type        = string
-  description = "The name of a new or an existing resource group in which to provision Secrets Manager resources to. If prefix input variable is passed then it will get prefixed infront of the value in the format of '<prefix>-value'. Optional if existing_secrets_manager_crn is not set."
+  description = "The name of a new or an existing resource group where the Secrets Manager resources are provisioned. If prefix input variable is passed then it is prefixed in front of the value in the format of '<prefix>-value'. Optional if existing_secrets_manager_crn is not set."
   default     = null
 }
 
 variable "region" {
   type        = string
-  description = "The region in which to provision Secrets Manager resources."
+  description = "The region where Secrets Manager resources are provisioned."
   default     = "us-south"
 }
 
 variable "prefix" {
   type        = string
-  description = "(Optional) Prefix to append to all resources created by this solution."
+  description = "An optional prefix to append to all resources created by this solution."
```

```suggestion
  description = "The prefix to apply to all resources created by this solution."
```

Trying to remove the optional label. It's clear in the Cloud UI.